cmd/elf2c/elf2c.go

```diff
@@ -136,6 +158,17 @@ func main() {
 		fmt.Fprintf(os.Stderr, "gencode\n")
 		_, file := path.Split(n)
 		fmt.Fprintf(w, "uintptr_t %v_main = %v;\n", n, f.Entry)
+		var msg string
+		if codestart == -1 {
+			msg = fmt.Sprintf("codestart was never set for %s; the ELF file has no 'R E' segment.", file)
+		}
+		if datastart == -1 {
+			msg = fmt.Sprintf("datastart was never set for %s; the ELF file has no 'RW' segment", file)
```

Comment:
Would it make sense to concatenate those for full feedback? Otherwise, half of the information may be lost of both aren't set.

Reply:
yeah, they were supposed to be, I forgot the +